src/adapter/src/coord/consistency.rs

```diff
@@ -41,6 +46,10 @@ impl Coordinator {
             inconsistencies.read_capabilities = read_capabilities;
         }
 
+        if let Err(active_webhooks) = self.check_active_webhooks() {
+            inconsistencies.active_webhooks = active_webhooks;
```

Comment:
Uncovered

Reply:
It's actually good that this is uncovered! This is part of a post-test consistency check that asserts the Coordinator state is correct. If this code was hit would mean we have an inconsistency somewhere